packages/realm/src/OrderedCollection.ts

```diff
@@ -748,4 +748,12 @@ export abstract class OrderedCollection<T = unknown, EntryType extends [unknown,
       return DEFAULT_COLUMN_KEY;
     }
   }
+
+  /**
+   * This symbol was required for implementing ReadOnlyArray, but we don't need
+   * it for anything.  It's just here to make TypeScript happy.
```

Comment:
@takameyer I don't think this is relevant as the public facing documentation though.

Reply:
Appears this was reverted anyway.  I'll look into why

Reply:
[Reverted](https://github.com/realm/realm-js/pull/5992#event-9822726832) this prior to the RC2 release due [enumeration errors](https://github.com/realm/realm-js/actions/runs/5528893972/jobs/10086388396). Have not had time to investigate further, but you can refer to the JS Slack thread 👍

Reply:
Very curious.  Thanks for the link.  Quite interesting that it only effects certain engines and platforms.  Perhaps it's binding related.  Node seems to be fine